SpiderBot.py

```diff
@@ -20,32 +21,49 @@
 logger.addHandler(handler)
 
 
+# Grab token from arguments
```

Comment:
you probably don't want to have your token in the arguments, as it would appear in the processes list and therefore could probably be leaked if a single account on the server gets taken over. You should use a configuration file for that instead.